webui/react/src/e2e/fixtures/global-fixtures.ts

```diff
@@ -88,10 +88,26 @@ export const test = baseTest.extend<CustomFixtures, CustomWorkerFixtures>({
     },
     { scope: 'worker' },
   ],
-  dev: async ({ page }, use) => {
-    const dev = new DevFixture(page);
+  // eslint-disable-next-line no-empty-pattern
+  dev: async ({}, use) => {
+    const dev = new DevFixture();
     await use(dev);
   },
+  devSetup: [
```

Comment:
now we automatically use a dev command on the UI for each page we get. the dev command is set to local storage